IDEMPIERE-6409 Charts not rendered on certain conditions

saulojg opened the pull request:
https://idempiere.atlassian.net/browse/IDEMPIERE-6409

uthadehikaru commented:
hi @saulojg 

please provide all pull request template message and check list the task.

thanks.

saulojg commented:
# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [x] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
Does not apply.

uthadehikaru reviewed (approved):
tested
